charts/data-dashboard-backend/README.md

```diff
@@ -33,7 +33,6 @@ API for data in the data dashboard
 | replicaCount | int | `1` | Number of replicas to deploy |
 | image.repository | string | `"radarbase/radar-data-dashboard-backend"` | docker image repository |
 | image.pullPolicy | string | `"Always"` | image pull policy |
-| image.tag | string | `nil` | Overrides the image tag whose default is the chart appVersion. |
```

Comment:
Do you know why this is being removed?

Reply:
Sorry, forgot to regenerate the docs. Entry is back now.